sdk/security/keyvault/azcertificates/example_test.go

```diff
@@ -94,3 +94,46 @@ func ExampleClient_DeleteCertificate() {
 	// In a soft-delete enabled vault, deleted resources can be recovered until they're purged (permanently deleted).
 	fmt.Printf("Certificate will be purged at %v", *resp.ScheduledPurgeDate)
 }
+
+func ExampleClient_ImportCertificate_pfx() {
+	// This example uses `ImportCertificate` to import a PFX certificate.
+
+	// Assuming you already have a PFX containing your key pair, you can import it into Key Vault.
+	// You can do this without setting a policy, but the policy is needed if you want the private key to be exportable
+	// or to configure actions when a certificate is close to expiration.
+	parameters := azcertificates.ImportCertificateParameters{
+		Base64EncodedCertificate: to.Ptr("<TODO: pfx cert value>"),
```

Comment:
Hmm I'd probably show reading bytes from a .pfx and encoding them. And I suppose the default `ContentType` is x-pkcs12 or similar, so it isn't strictly necessary to set it in this case? I'd set it anyway so readers know about it.

Reply:
fixed in [ff9290f](https://github.com/Azure/azure-sdk-for-go/pull/23123/commits/ff9290f757c8c0b1189bef2823d87c3bc2b33275)